chrome/installer/linux/common/installer.include

```diff
@@ -220,7 +220,7 @@ stage_install_common() {
 
   # Inference Engine
   if [ -f "${BUILDDIR}/libinference_engine.so" ]; then
-    for file in libinference_engine.so libMKLDNNPlugin.so libclDNNPlugin.so libGNAPlugin.so libgna_api.so libgna_kernel.so; do
+    for file in libinference_engine.so libMKLDNNPlugin.so libclDNNPlugin.so libGNAPlugin.so libgna_api.so libgna_kernel.so libSRC.so; do
```

Comment:
Please make sure it can run on Windows after packaging the binary with "ninja -C out\Release chrome mini_installer"